company/values.md

```diff
@@ -1,15 +1,34 @@
 # Sourcegraph values
 
-Our values are:
+Our values reflect who we are and how we work.
 
-## People
+## People first
 
-Together we are advancing technology for the good of people all around the world. We will attract, hire and retain the best teammates in the world and treat everyone in a first-class manner.
+[Our amazing team](teammate_qualities.md) is what makes Sourcegraph special. Together we are advancing technology for the good of people all around the world and we treat everyone in a first-class manner.
```

Comment:
It's unclear to me what the difference is between company values and team mate qualities. Is there room to unify these?

Reply:
If I had to guess, I'd say that when you combine them into one, you might lose clarity, since some individual traits might start so sound really wishy-washy when raised to the company level.